### PR TITLE
Updates documentation for Console release 0.20.0

### LINKED ISSUE #535: [RELEASE] Console 0.20.0
The [MinIO Console 0.20.0](https://github.com/minio/console/releases/tag/v0.20.0) release has changes that impact the docs:

- [x] Console Support's registration page has been updated with tabs ([#2250](https://github.com/minio/console/pull/2250))
- [x] Metric data can be downloaded as PNG or CSV ([#2241](https://github.com/minio/console/pull/2241))
- [x] Users can now delete configured notification targets in the console ([#2213](https://github.com/minio/console/pull/2213))

### PR DESCRIPTION
**Pull request by djwfyi**
Closes #535

- Updates MinIO Console doc section on the Support tab
- Adds info to several pages that Prometheus metrics displayed in the console can now be downloaded
- Adds that bucket notification targets can now be deleted from the console
- Updates the MinIO Console page to reflect the current navigation layout by moving around sections and updating screenshots

### CODE REVIEW

File: source/operations/monitoring.rst [hunk 1]
```diff
@@ -28,9 +28,11 @@ collected metrics.
   trigger alerts based on collected metrics. See
   :ref:`minio-metrics-and-alerts-alerting` for more information.
 
-MinIO publishes collected metrics data using Prometheus-compatible data
-structures. Any Prometheus-compatible scraping software can ingest and
-process MinIO metrics for analysis, visualization, and alerting.
+When configured, the :ref:`MinIO Console <minio-console-metrics>` shows some metrics in the :guilabel:`Monitoring > Metrics` page.
```
Comment (ravindk89) [inline source/operations/monitoring.rst:31]: So talking to AB about this, our distinction is this:

- MinIO by default provides point-in-time metrics on cluster status and operations.
- For deployments configured to use Prometheus, the Console can provide time-series data on an additional set of metrics.

File: source/operations/monitoring.rst [hunk 2]
```diff
@@ -28,9 +28,11 @@ collected metrics.
...
+You can download these metrics as either ``.png`` images or ``.csv`` files.
+
+MinIO publishes collected metrics data using Prometheus-compatible data structures.
```
Comment (ravindk89) [inline source/operations/monitoring.rst:34]: Move this up to the top and I'll steal/modify this as part of my ongoing metrics/monitoring reworks.

**Review by ravindk89 — APPROVED** [review]
Two requested changes, otherwise LGTM

File: source/operations/monitoring.rst (hunk 1 above)
Comment (djwfyi) [reply source/operations/monitoring.rst:31]: I reworked this section to incorporate this feedback.

File: source/operations/monitoring.rst (hunk 2 above)
Comment (djwfyi) [reply source/operations/monitoring.rst:34]: Done.

**Pull request merged**